### PR TITLE
added wallpapers from 500px

### PR DESCRIPTION
**Pull request by theclue**
These images come from royalty-free photos I own the (c).

If you think they would fit well with Haiku, I'll be proud and happy to "license" to Haiku for free for use as wallapers and other marketing or communication purposes. If you need in higher resolutions for ex. 4K, retina, etc., just ask.

Hope you like them!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 target
+.DS_Store
+.DS_Store
```
Comment (waddlesplash) [inline .gitignore]: You only need one of these lines

**Comment by theclue** [comment]
here we go :)

**Comment by kallisti5** [comment]
sounds good to me :-)

We'll need to add another package for wallpaper_500px or something to Haikuports... but this gives us some nice options.

Thanks!

**Comment by kallisti5** [comment]
oh.. small note.  Gut reaction is MIT allows commercial use... so by stating not available for commercial use means we can't ship the 500px wallpapers with a Haiku release....

...but then again, technically no 3rd parties can "sell" a Haiku release since our registered + trademarked logo is in the release... and you gave full access to Haiku, Inc. so it should be fine.

**Comment by theclue** [comment]
> oh.. small note. Gut reaction is MIT allows commercial use... so by stating not available for commercial use means we can't ship the 500px wallpapers with a Haiku release....

if needed, i could re-release under MIT. But then i will have to put down from my 500px store. I'm not that those few bucks per months are going to change my life but...if i can avoid that... :)

**Pull request merged**